spyderlib/plugins/editor.py

```diff
@@ -626,6 +626,14 @@ def get_plugin_actions(self):
         add_shortcut_to_tooltip(self.new_action, context="Editor",
                                 name="New file")
         
+        self.open_last_closed = create_action(self, _("O&pen last closed"),
+                icon=ima.icon('filenew'), tip=_("Open last closed"),

```

Comment:
Please remove the  `filenew` icon from this action. Opening the last closed file is not the same as opening a new one, so we use the same icon for both, that'd confuse our users.